modules/@apostrophecms/schema/ui/apos/components/AposInputArray.vue

```diff
@@ -1,92 +1,62 @@
-          tag="div"
-          role="list"
-          :list="items"
-          v-bind="dragOptions"
-          :id="listId"
-        >
-          <div
-            v-for="(item, index) in items"
-            :key="item._id"
-            class="apos-input-array-inline-item"
-            :class="item.open ? 'apos-input-array-inline-item--active' : null"
-          >
-            <div class="apos-input-array-inline-item-controls">
-              <AposIndicator
-                icon="drag-icon"
-                class="apos-drag-handle"
-              />
-              <AposButton
-                v-if="item.open && !alwaysExpand"
-                class="apos-input-array-inline-collapse"
-                :icon-size="20"
-                label="apostrophe:close"
-                icon="unfold-less-horizontal-icon"
-                type="subtle"
-                :modifiers="['inline', 'no-motion']"
-                :icon-only="true"
-                @click="closeInlineItem(item._id)"
-              />
-            </div>
-            <div class="apos-input-array-inline-content-wrapper">
-              <h3
-                class="apos-input-array-inline-label"
-                v-if="!item.open && !alwaysExpand"
-                @click="openInlineItem(item._id)"
+        <table class="apos-table" v-if="items.length">
+          <thead>
+            <tr class="apos-input-array-inline-header">
+              <th class="apos-table-head--hidden" />
+              <th
+                v-for="schema in field.schema"
+                :key="schema._id"
+                class="apos-table-head"
               >
-                {{ getLabel(item._id, index) }}
-              </h3>
-              <transition name="collapse">
-                <div
-                  v-show="item.open"
-                  class="apos-input-array-inline-schema-wrapper"
-                >
-                  <AposSchema
-                    :schema="field.schema"
-                    v-model="item.schemaInput"
-                    :trigger-validation="triggerValidation"
-                    :utility-rail="false"
-                    :generation="generation"
-                    :modifiers="['small', 'inverted']"
-                    :doc-id="docId"
-                  />
-                </div>
-              </transition>
-            </div>
-            <div class="apos-input-array-inline-item-controls apos-input-array-inline-item-controls--remove">
-              <AposButton
-                label="apostrophe:removeItem"
-                icon="trash-can-outline-icon"
-                type="subtle"
-                :modifiers="['inline', 'danger', 'no-motion']"
-                :icon-only="true"
-                @click="remove(item._id)"
-              />
-            </div>
-          </div>
-        </draggable>
+                {{ $t(schema.label) }}
+              </th>
+            </tr>
+          </thead>
+          <tbody class="apos-input-array-inline">
+            <tr
```

Comment:
Draggable on this list of tr's would make sense (when not disabled for the field).